templates/event/event_show.html.twig

```diff
@@ -11,49 +11,74 @@
+        {% else %}
+            <img class="img-fluid event-img mr-3 my-2"  src="{{ asset("images/rolendar-1.jpg") }}" alt="{{ event.name }}" align="left">
+        {% endif %}
 
-        {%  if event.status == "cancelled" %}
+        {% if event.status == constant('STATUS_CANCELLED', event) %}
             <h4 class="p-2">
-                <span class="badge badge-danger">Cancelled!</span>
+                <span class="badge badge-danger">Отменено!</span>
             </h4>
         {% endif %}
 
-        <h4 class="text-muted p-2">
+        <h4 class="text-muted pl-1">
             {{ event.startDate | date("d/m/Y") }}
             {% if event.startDate != event.endDate %}
                  - {{ event.endDate | date("d/m/Y") }}
             {% endif %}
-            <span class="badge badge-primary float-right brand-bg"> {{ event.type.name | upper }} </span>
         </h4>
-        <h4 class="text-muted p-2">
+        <h4 class="text-muted pl-1">
             {% if event.location %}
                 {{ event.location }},
             {% endif %}
             {{ event.region.name }}
-            {% if event.subgenres %}
-                <span class="badge bg-white float-right ">
-                    {% for subgenre in event.subgenres%}
-                        {% if loop.last %}
-                            {{ subgenre.name }}
-                        {% else %}
-                            {{ subgenre.name }},
-                        {% endif %}
-                    {% endfor %}
-                </span>
-            {% endif %}
         </h4>
 
-        <p class="p-2">{{event.description}}</p>
+        {% if event.type %}
+            <p class="pl-1"> <strong> Формат:</strong> {{ event.type.name}} </p>
+        {% endif %}
+
+        {% if event.genre %}
+            <p class="pl-1"> <strong> Жанр:</strong> {{ event.genre.name }} </p>
+        {% endif %}
+
+        {% if event.subgenres %}
+            <p class="pl-1"> <strong> Поджанры: </strong>
+                {% for subgenre in event.subgenres%}
+                    {% if loop.last %}
+                        {{ subgenre.name }}
+                    {% else %}
+                        {{ subgenre.name }},
+                    {% endif %}
+                {% endfor %}
+            </p>
+        {% endif %}
 
         {% if event.organizers %}
-            <p class="p-1"> <strong> Organizers:</strong> {{ event.organizers }} </p>
+            <p class="pl-1"> <strong> Организаторы:</strong> {{ event.organizers }} </p>
+        {% endif %}
+
+        {% if event.settlement %}
+            <p class="pl-1"> <strong> Тип поселения:</strong> {{ event.settlement.name }} </p>
+        {% endif %}
+
+        {% if not event.weapons is empty  %}
+            <p class="pl-1"> <strong> Материал оружия: </strong>
+                {% for weapon in event.weapons%}
```

Comment:
@daria-sieroshtan 
pls pay attention to spaces before / after twig tags, ut's absense breaks syntax highlight